frontend/src/lib/k8s/node.ts

```diff
@@ -51,7 +51,15 @@ class Node extends makeKubeObject<KubeNode>('node') {
     const [nodeMetrics, setNodeMetrics] = React.useState<KubeMetrics[] | null>(null);
     const [error, setError] = useErrorState(setNodeMetrics);
 
-    useConnectApi(metrics.bind(null, '/apis/metrics.k8s.io/v1beta1/nodes', setNodeMetrics, setError));
+    function setMetrics(metrics: KubeMetrics[]) {
+      setNodeMetrics(metrics);
+
+      if (metrics !== null) {
```

Comment:
Above(in the setList) we have cuddled the if with the previous statement i think we should follow the same here to have consistency. And also same nitpick this can be reduced to
```suggestion
      if (metrics) {
```